internal/common/types.go

```diff
@@ -0,0 +1,126 @@
+package store
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+)
+
+// ParseCustomTypeError is thrown when parsing strings into enumerations
+type ParseCustomTypeError struct {
+	reason string
+}
+
+func (err *ParseCustomTypeError) Error() string {
+	return err.reason
+}
+
+// Architecture represents one of the supported CPU architectures available for images
+// produced by osbuild-composer. I decided to represent it as an integer because if it
```

Comment:
Use the passive voice in comments: "It is represented as an integer because...". The same for the rest of the comments. This doesn't make a difference to the meaning of course, just keeps the style consistent.

Reply:
Sure. It's just that all the grammar-checking tools nowadays complain about overuse of passive voice, that I'm trying to avoid it from time to time.